### PR TITLE
chore: add get res_name && resource_type function, and append home dir path in log dir

### PR DESCRIPTION
**Pull request by fanhan**
I use sentinel with myself config, then i need res_name && resource_type

* add get res_name function
* add get resource_type function
* append home dir path in log dir

### CODE REVIEW

**Comment by jjeffcaii** [comment]
@Forsworns Could please review this PR? We need some fields which existing in Golang SDK, but it seems missed in the Rust SDK, this PR can help to resolve such problems. Thanks! ❤️

File: sentinel-core/src/core/config/base.rs [hunk 1]
```diff
@@ -244,3 +244,11 @@ pub fn metric_stat_sample_count() -> u32 {
         .try_with(|c| c.borrow().config.stat.sample_count)
         .unwrap()
 }
+
+#[inline]
+pub fn get_default_log_dir() -> String {
+    match dirs::home_dir() {
+        Some(path) => path.join(LOG_METRICS_DIR).to_string_lossy().to_string(),
```
Comment (Forsworns) [inline sentinel-core/src/core/config/base.rs:251]: I think the default behavior should be the same as the [Sentinel](https://github.com/alibaba/Sentinel) and [sentinel-golang](https://github.com/alibaba/sentinel-golang). If you need a different log directory, you can set it with a configuration entity or a configuration file, see these examples:

https://github.com/sentinel-group/sentinel-rust/blob/main/examples/config/from_yaml/from_yaml.rs
https://github.com/sentinel-group/sentinel-rust/blob/main/testdata/config/sentinel.yaml
or 
https://github.com/sentinel-group/sentinel-rust/blob/main/examples/config/from_entity/from_entity.rs

Comment (Forsworns) [reply sentinel-core/src/core/config/base.rs:251]: @sczyh30 @LearningGp If you think this minor difference is acceptable, I have no other concerns.

Comment (jjeffcaii) [reply sentinel-core/src/core/config/base.rs:251]: @Forsworns But, it seems that Golang SDK use `homedir` as its default path, please see this link: https://github.com/alibaba/sentinel-golang/blob/08071855bc67423777353c6e02f1390b419172b1/core/config/config.go#L178
I'm not familiar with sentinel source codes, please confirm, thanks!

Comment (Forsworns) [reply sentinel-core/src/core/config/base.rs:251]: @jjeffcaii I see, I forget why this is different from the sentinel-golang, it should not have been.

**Pull request merged**